.yarn/patches/app-builder-lib-npm-24.13.3-86a66c0bf3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/templates/nsis/include/allowOnlyOneInstallerInstance.nsh b/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
```

Comment:
Thanks for fixing this. Could you please add a comment here to explain what it does and why it's there (and linking to this PR for example). You can add comments with the `#` prefix